test/common/schemaform/widgets/SelectWidget.unit.spec.jsx

```diff
@@ -122,4 +122,24 @@ describe('Schemaform <SelectWidget>', () => {
     });
     expect(onBlur.calledWith('testing', 'test')).to.be.true;
   });
+  it('should not render blank option when default exists', () => {
```

Comment:
I think it's covered ok by the first test, which requires there be two options for an enum with a single value.